pkg/flowaggregator/flowaggregator.go

```diff
@@ -217,28 +298,27 @@ func (fag *flowAggregator) Run(stopCh <-chan struct{}) {
 	}
 }
 
-func (fag *flowAggregator) sendFlowKeyRecords(key ipfixintermediate.FlowKey, records []ipfixentities.Record) error {
+func (fag *flowAggregator) sendFlowKeyRecords(key ipfixintermediate.FlowKey, record ipfixintermediate.AggregationFlowRecord) error {
 	setSent := ipfix.NewSet(ipfixentities.Data, fag.templateID, false)
-	for _, record := range records {
-		elements := make([]*ipfixentities.InfoElementWithValue, 0)
-		for _, ieString := range fag.infoElementsOrder {
-			element, exist := record.GetInfoElementWithValue(ieString)
-			if !exist {
-				return fmt.Errorf("%s not present.", ieString)
-			}
-			elements = append(elements, element)
-		}
-		err := setSent.AddRecord(elements, fag.templateID)
-		if err != nil {
-			return fmt.Errorf("AddRecord failed, error: %v", err)
+	elements := make([]*ipfixentities.InfoElementWithValue, 0)
```

Comment:
could you add a check to see if record.ReadyToSend true or not? This will make sure that the record is correlated and have both records from source and destination.

Reply:
done. Thanks!